acquia/mc-cs-plugin-custom-objects
# Maut 2896

https://backlog.acquia.com/browse/MAUT-2896

#### Description:
This PR adds ability to show more details in lists on contact and custom item detail pages.

#### Steps to test this PR:
1. Create some custom object items with some fields keeping `Show in a custom object detail list` and `Show in a contact detail list` on their defaults (`= Yes`).
2. Create some links for both custom items and contacts.
3. Go to a detail page of custom item that was linked items created for.
4. You should see the following columns in the list: `Name`, `ID` + `all the fields you created in the first step`.
5. Set `Show in a custom object detail list` to value `No` for some fields.
6. You should no longer see the given columns in that list.
7. Repeat the steps 3. to 6. but for contacts with the `Show in a contact detail list` flag.
